packages/node/src/plugins/LoadFileChunkLoadingRuntimeModule.ts

```diff
@@ -251,7 +256,11 @@ class ReadFileChunkLoadingRuntimeModule extends RuntimeModule {
         : '// no remote script loader needed',
       withLoading
         ? Template.asString([
-            '// ReadFile + VM.run chunk loading for javascript',
+          new DynamicFileSystem().generate(),
```

Comment:
Keeping runtime modules decoupled from implementation. As such, i just call generate() for module in module.